Sample Data/intel471_titan_API_malware_indicators.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
@mmolenda, sample data file must be in array, please this [file ](https://github.com/Azure/Azure-Sentinel/blob/master/Sample%20Data/CitrixAnalytics_userProfile_CL.json)for reference